### PR TITLE
Test/user sign in

### PR DESCRIPTION
**Pull request by cvillamer13**
Add user login and registration

### CODE REVIEW

File: src/utils/openai.util.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import openai from '../configs/openai.config'
 import dotenv from 'dotenv'
-
+console.log(openai);
```
Comment (aldenvallestero) [inline src/utils/openai.util.ts:3]: Please remove this console.log() because we are not going to use this.

File: src/services/user.service.ts [hunk 2]
```diff
@@ -61,12 +64,16 @@ export async function signIn(data: ISignIn) {
...
+      
+
+      const DataRes = JSON.stringify(result)
+      const DataRes2 = JSON.parse(DataRes)
       return {
         status: 200,
         data: {
-          userID: result.user.uid,
-          email: result.user.email,
+          accessToken: DataRes2.user.stsTokenManager.accessToken,
```
Comment (aldenvallestero) [inline src/services/user.service.ts:76]: Try removing the line 71-72 and from this line 76, try:
result.stsTokenManager.accessToken

Comment (cvillamer13) [reply src/services/user.service.ts]: Ahhhhhh ganun pala yun kasi nung nag consle.log ako result.users.stsTokenManager ayaw gumana pero under ng users yun sige sige

File: src/utils/openai.util.ts (hunk 1 above)
Comment (cvillamer13) [reply src/utils/openai.util.ts]: Copy sir

**Review by cvillamer13 — COMMENTED** [review]
@aldenvallestero ok na na change ko na pati na remove na console.log

**Review by aldenvallestero — APPROVED** [review]

**Pull request merged**